src/repl.js

```diff
@@ -96,21 +97,34 @@ Prototype REPL - https://github.com/nodejs/repl`,
     const evaluateResult = await this.eval(line, awaited);
 
     if (evaluateResult.exceptionDetails) {
+      // lets try for recovering
+      const result = isRecoverableError(evaluateResult.exceptionDetails.exception, line);
+      if (result) {
+        this.io.setPrefix('...');
```

Comment:
io should set the prefix when it receives $multiline. also the `...` needs a space after.